lib/drivers/bayer/bayerContourNext.js

```diff
@@ -149,7 +150,11 @@ module.exports = function (config) {
   var parseHeader = function (header, callback){
     var pString = header.split('|');
     var pInfo = pString[4].split('^');
-    var sNum = pInfo[2].match(/^\d+\-\s*(\w+)/);
+
+    // serial number can be in two formats:
+    // 6301-1C2CF8C -> C2CF8C
```

Comment:
This would be 1C2CF8C with the original regex.

Reply:
Hmm, I decided to upload all 11 Bayer meters I have here and my "bug fix" won't work, as that one is the odd one out. I'll explain below in response to the regex comment.